library/build.gradle.kts

```diff
@@ -6,6 +6,7 @@ plugins {
 }
 
 group = "com.github.vinted"
+version = "0.4.2"
```

Comment:
Update version in ```Dependencies``` file: https://github.com/vinted/actioncable-client-kotlin/blob/master/buildSrc/src/main/kotlin/Dependencies.kt#L18

Also, there is a version number in Readme file ```Download``` section. Probably we should remove it from there as it is very easy to forget to update the number there.

Reply:
Actually I should remove this version from PR.